src/main.py

```
@@ -115,7 +115,7 @@ async def lifespan(_: FastAPI):
     title="Honcho API",
     summary="The Identity Layer for the Agentic World",
     description="""Honcho is a platform for giving agents user-centric memory and social cognition""",
```

Pylint (3.3.7)

[convention] Line too long (102/100)

(C0301)